code/game/objects/effects/decals/Cleanable/aliens.dm

```diff
@@ -57,12 +60,21 @@
 /obj/effect/decal/cleanable/xenoblood/xgibs/body
 	random_icon_states = list("xgibhead", "xgibtorso")
 
+/obj/effect/decal/cleanable/xenoblood/xgibs/torso
+	random_icon_states = list("xgibtorso")

```

Comment:
Since gibbing can drop arms, legs, head, but never chest, we make a specific torso gib to always be spawned on gibbing.